### PR TITLE
Update README to list required request headers

### PR DESCRIPTION
**Pull request by gerpsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +74,7 @@ Other options that are supplied are passed query options if they are known, othe
 
 - Only `SELECT` statements are supported.
 - Statements using parameters must use the `:param` syntax and must have a corresponding entry in the `params` map.
-- The only standard 
+- The only standard
```
Comment (gerpsh) [inline README.md]: What was this line meant to be?

Comment (bruth) [inline README.md]: No idea. I should remove it.

Comment (bruth) [inline README.md]: The change came from my editor that trims whitespace.

Comment (bruth) [inline README.md]: Or your editor..

Comment (gerpsh) [inline README.md]: No I mean it looks incomplete, was there something else to the bullet point?

Comment (bruth) [inline README.md]: It is incomplete, but I don't remember what I was typing, so feel free to remove it.

**Pull request merged**